docs/api-server-walkthrough.md

```diff
@@ -104,7 +104,7 @@ kubectl needs a basic kubeconfig setup. A kubeconfig consists of three
 sections that can be set with the following three commands:
 
 1. `kubectl config set-credentials service-catalog-creds --username=admin --password=admin`
-1. `kubectl config set-cluster service-catalog-cluster --server=https://localhost:6443 --certificate-authority=/var/run/kubernetes/apiserver.crt`
+1. `kubectl config set-cluster service-catalog-cluster --server=https://localhost:6443 --certificate-authority=/var/run/kubernetes-service-catalog/apiserver.crt`
```

Comment:
I need to rely on this rather than configuring the kubectl command on each run below.